python/plugins/db_manager/dlg_import_vector.py

```diff
@@ -51,6 +52,11 @@ def __init__(self, inLayer, outDb, outUri, parent=None):
         self.outUri = outUri
         self.setupUi(self)
 
+        objClass = self.db.searchClass()
+        if objClass != "PGDatabase":
```

Comment:
This should not have any knowledge about the database behind. Rather call a method that asks the database if it supports comments.

Like `self.db.supportComments()`

Reply:
Oh well yes that's a good idea